ovos_utils/gui.py

```diff
@@ -49,6 +49,15 @@ def can_use_gui(bus=None, local=False):
         return can_use_local_gui()
     return can_use_local_gui() or is_gui_connected(bus)
 
+def extend_about_data(about_data):
+    bus = get_mycroft_bus()
```

Comment:
usually this should be avoided when possible to not create a new websocket connection, can you add `bus=None`to the arguments and make this `bus = bus or get_mycroft_bus()`? that allows to re utilize a open websocket which will already exist in basically everything that will consume this new method

Reply:
done